src/js/components/Button/StyledButtonKind.js

```diff
@@ -1,6 +1,7 @@
 import styled, { css } from 'styled-components';
 
 import {
+  activeStyle,
```

Comment:
We found that ActiveStyle was not pulled over from StyledButton.js so it was not getting the same behavior.